logcat/src/jvmMain/kotlin/logcat/Throwables.kt

```diff
@@ -0,0 +1,20 @@
+package logcat
+
+import java.io.PrintWriter
+import java.io.StringWriter
+
+/**
+ * Utility to turn a [Throwable] into a loggable string.
+ *
+ * The implementation is based on Timber.getStackTraceString(). It's different
+ * from [android.util.Log.getStackTraceString] in the following ways:
+ * - No silent swallowing of UnknownHostException.
+ * - The buffer size is 256 bytes instead of the default 16 bytes.
+ */
+actual fun Throwable.asLog(): String {
```

Comment:
Seems duplicated?

Reply:
I was able to make `androidMain` depend on `jvmMain` and remove the duplication (even though based on https://github.com/square/logcat/pull/15#discussion_r876523744, it looked like sharing Android + JVM wasn't supported), and that appears to build & pass tests.  The IDE is showing unresolved references, though, so I'm working through that right now